html-forms/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+    <style>
+      ul {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="forms-submissions.js" method="post">
+      <ul>
+        <li>
+          <label for="email">Email</label>
+        </li>
+        <li>
+          <input type="email" id="email" name="user_email" />
+        </li>
+        <li>
+          <label for="coding">Why is coding cool?</label>
+        </li>
+        <li>
+          <select id="coding" name="coding_opinion">
+            <option>Select an option</option>
+            <option>Because it's interesting</option>
+            <option>Because it's fun</option>
+            <option>Because it's powerful</option>
+          </select>
+        </li>
+        <li><p>Favorite Language (choose one)</p></li>
+        <li>
+          <input type="radio" id="html" name="favorite_language" value="HTML" />
+          <label for="html">HTML</label>
+          <input type="radio" id="CSS" name="favorite_language" value="CSS" />
+          <label for="CSS">CSS</label>
+          <input
+            type="radio"
+            id="JavaScript"
+            name="favorite_language"
+            value="JavaScript" />
+          <label for="JavaScript">JavaScript</label>
+        </li>
+        <li><p>Would you like to sign up for more information?</p></li>
+        <li>
+          <input
+            type="checkbox"
+            id="signup"
+            name="sign_up"
+            value="signedup" /><label for="signup">Sign me up!</label>
+        </li>
+        <li><label for="comments">Comments</label></li>
+        <li><textarea cols="30" rows="8"></textarea></li>
```

Comment:
This `textarea` should have an `id` of "comments" to match the `for` of your `label`

Reply:
`        <li><textarea id="comments" cols="30" rows="8"></textarea></li>`

Added